### DISCUSSION TITLE
2차 팀 회고

### DISCUSSION THREAD

**Discussion by 00TaciTa00** [codeit-fe2-2/taskify#56] [Q&A]
양식 복사해서 적어주세요~
# Keep
- 이거 좋았어요!
# Problem
- 이런 문제점은 고치고 싶어요...
# Try
- 이런 거 해보고 싶어요!

**Comment by ckswnskfk** [comment]
# Keep
- 리뷰가 전보다 활발해졌다! 감사하다!
# Problem
- 생산성이 좀 떨어지는 것 같다.
- 코드 일관성이 떨어지는 것 같다.
# Try
- 질문을 편하게 하자! 멘토님께도 여쭙자!
- 코드 컨벤션(네이밍, 타입 등?)을 더 명확히 하자(여력이 되면..)

**Comment by 00TaciTa00** [comment]
# Keep
- 리뷰 빈도가 늘었습니다!
# Problem
- 리뷰 안 하시는 분도 해주셨으면 좋겠어요... ㅎㅎ 하시는 분들만 하셔서
# Try
- 우리... 코어 타임에는 같이 음성 채팅방에 있을래요...? 같이 진행상황 실시간으로 얘기하며 작업해요...

**Comment by caseV12** [comment]
# Keep
- 리뷰를 꼼꼼히 해주셔서 배워가는 점이 많네요
- 디코에서 PR이랑 머지할 때마다 잘 알려주셔서 좋았습니다.
# Problem
- 
# Try
-

**Comment by sung-eunji** [comment]
## Keep
- 'pr - merge' 과정에 규칙이 정해져서 깃 순환이 잘 됩니다.
- 비교적 자유로운 분위기: 서로가 조심스럽지만 솔직하게 상태를 공유하고 있는 분위기가 좋습니다.
## Problem
- 
## Try
- 압박 웰컴입니다 ..!

**Comment by BBoBBoM** [comment]
# Keep
- 회의록작성 해야 할일에 대한 구분이 명확해 작업대기부분에서 찾아서 작업하면 되는부분이 좋다.
- 본인의 상황 진도 상황에 대한 공유가 확실함
# Problem
- 컴포넌트 작성시 서로 작성하는 스타일에 대한 차이 부분 (이벤트 처리 prop 상태 관리에 대한) 의견 부족
- 맡은 작업의 진행 속도가 팀에 비해 느리다
# Try
- PR 확인 적극적으로, 리뷰 달기하겠습니다.
- 원활한 작업을 위해, 부족한 부분 공부를 더 해야할 것 같다.
